### DISCUSSION TITLE
Invoke a recipe from another file as a dependency

### DISCUSSION THREAD

**Discussion by bhollis** [casey/just#2009] [Ideas]
As we expand our usage of `just`, we'd like to be able to move recipes between modules in our repo. I know we can import rules between files, but it'd be great if we could simply *invoke* rules in other files. e.g.:

`./justfile`:
```
foo: bar/baz
  echo "this happens after bar's baz recipe"
```

`./bar/justfile`:
```
baz:
  echo "it's baz time!"
```

I can run `just bar/baz` at the CLI and it'll work, but declaring `bar/baz` as a dependency of `foo` doesn't work the same way:

```
$ just bar/baz
echo "it's baz time!"
it's baz time!
$ just foo
error: Expected '&&', comment, end of file, end of line, identifier, or '(', but found '/'
 ——▶ justfile:1:9
  │
1 │ foo: bar/baz
  │         ^
```

**Comment by casey** [comment]
This is desired, but not yet implemented. See #929.
